Sources/TensorFlow/Loss.swift

```diff
@@ -290,3 +290,30 @@ public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
     let negAbsLogits = max(logits, -logits) // Custom `abs` to compute gradients at `0`.
     return reduction(maxLogitsWithZero - logits * labels + log1p(exp(-negAbsLogits)))
 }
+
+/// Returns the Huber loss between predictions and expectations.
+///
+/// For each value `x` in the difference `exprected - predicted`, the loss equals `0.5 * x^2` if
```

Comment:
```suggestion
/// For each value `x` in the difference `expected - predicted`, the loss is:
/// - `0.5 * x^2` if `abs(x) <= δ`.
/// - `0.5 * δ^2 + δ * (|x| - δ)` otherwise.
```